README.md

```diff
@@ -108,6 +108,18 @@ Here are a few examples:
   $ python -m main +experiment=simple_dqn/connect_four +run.hill_training_trials_ratio=0.05
   ```
 
+- Launch a [DAGGER](https://arxiv.org/abs/1011.0686) imitation learning algorithm by first training the expert using the simple_a2c method
```

Comment:
Can you create a dedicated "page" for DAGGER (in docs/results) that have this exact content and maybe some further details and training examples.